Composer/packages/server/src/models/utilities/util.ts

```diff
@@ -40,5 +40,11 @@ export const getRemoteFile = async (url): Promise<string> => {
 
 // convert zip folder name to skill name
 export const convertFolderNameToSkillName = (path, skillName) => {
```

Comment:
suggestion use `path` module to operate with paths. Not tested but something like this should work:
```js
export const convertFolderNameToSkillName = (pathStr, skillName) => {
  const folderName = path.parse(pathStr).dir;
  const relativePath = path.relative(folderName, pathStr);
  
  return path.join(skillName, relativePath);
};
```